instrumentation/net/http/otelhttp/internal/semconv/v1.20.0.go

```diff
@@ -144,7 +143,7 @@ func (o OldHTTPServer) MetricAttributes(server string, req *http.Request, status
 
 	attributes := slices.Grow(additionalAttributes, n)
 	attributes = append(attributes,
-		standardizeHTTPMethodMetric(req.Method),
+		semconv.HTTPMethod(standardizeHTTPMethod(req.Method)),
```

Comment:
`semconv.HTTPMethod` is just an alias for string. We don't need to force that type here.

```suggestion
		standardizeHTTPMethod(req.Method),
```